python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -379,16 +382,21 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgInfoL
         @param tempExpRefList: List of data references to tempExp
         @param imageScalerList: List of image scalers
         @param weightList: List of weights
-        @param bgInfoList: List of background data from background matching
+        @param bgInfoList: List of background data from background matching, or None
+        @param altMaskList: List of alternate masks to use than those on disk, or None
```

Comment:
I'm having difficulty parsing that description of `altMaskList`. I think the English is wrong (I'm struggling here -- because you can't use "alternate" in this sense in British English -- but I think you want "alternate to" rather than "alternate than"). I guess that by "those on disk" you mean "those that were stored with the tempExp", right? The alternatives could be stored on disk elsewhere, one assumes. Could you clarify the wording please?